### PR TITLE
Add option to mirror local video feed

### PR DESCRIPTION
**Pull request by lukebarnard1**
Option that enables https://github.com/vector-im/riot-web/pull/5437

### CODE REVIEW

File: src/components/views/voip/VideoView.js [hunk 1]
```diff
@@ -108,14 +111,16 @@ module.exports = React.createClass({
                  document.mozFullScreenElement ||
                  document.webkitFullscreenElement);
         const maxVideoHeight = fullscreenElement ? null : this.props.maxHeight;
-
+        const localVideoFeedClasses = classNames("mx_VideoView_localVideoFeed",
+            {"mx_VideoView_localVideoFeed_flipped": UserSettingsStore.getSyncedSetting}
```
Comment (dbkr) [inline src/components/views/voip/VideoView.js]: did you want to actually call this function?

Comment (lukebarnard1) [reply src/components/views/voip/VideoView.js]: yes...

File: src/components/views/voip/VideoView.js [hunk 2]
```diff
@@ -112,7 +112,9 @@ module.exports = React.createClass({
                  document.webkitFullscreenElement);
         const maxVideoHeight = fullscreenElement ? null : this.props.maxHeight;
         const localVideoFeedClasses = classNames("mx_VideoView_localVideoFeed",
-            {"mx_VideoView_localVideoFeed_flipped": UserSettingsStore.getSyncedSetting}
+            { "mx_VideoView_localVideoFeed_flipped":
+                UserSettingsStore.getSyncedSetting('VideoView.flipVideoHorizontally'),
```
Comment (dbkr) [inline src/components/views/voip/VideoView.js]: nitpick, but would be good to specify default false as the default default is null

Comment (lukebarnard1) [reply src/components/views/voip/VideoView.js]: Fair!

**Pull request merged**